### ISSUE TITLE
Notebooks - Sparse Scipy Optim - syntax errors

### ISSUE THREAD

**Issue by thedatalass** [pysal/notebooks#59]
[sparse_scipy_optim](https://hub.gke.mybinder.org/user/darribas-notebooks-9rhfbljb/notebooks/docs/content/model/spint/sparse_scipy_optim.ipynb)

```
File "<ipython-input-11-adb8f934fbcb>", line 2
    print x
          ^
SyntaxError: Missing parentheses in call to 'print'. Did you mean print(x)?
```

**Comment by ljwolf** [comment]
These are all `2to3` errors, so not *technically* errors?

@darribas, maybe run 2to3 on these notebooks as part of the buildscript, or we just fix these once & move on.

**Comment by darribas** [comment]
yeah I could add that script run on the build process. Do you have any pointers on where to find it/how to run it?

More generally, shouldn't notebooks be Python-3 first in all projects in the federation by now? If so, I'm happy to issue a fix upstream on the notebooks.
